client/src/test/java/client/connection/control/ClientCommandEncoderTest.java

```diff
@@ -0,0 +1,44 @@
+package client.connection.control;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class ClientCommandEncoderTest {
+
+    @Test
+    public void shouldGetCorrectCodeWhenGetMessageForMoveRight() {
```

Comment:
ok. these tests make me uncomfortable hence ClientCommandEncoder class is not a prosperous. Lets consider refactoring ClientControlCommandType class to avoid them:

```
package client.connection.control;

public enum ClientControlCommandType {
    MOVE_LEFT(1),
    MOVE_RIGHT(2),
    MOVE_UP(3),
    MOVE_DOWN(4),
    SHOOT(5),
    START(6),
    EXIT(7),
    NONE(8);


    private final int code;

    private ClientControlCommandType(int code) {
        this.code = code;
    }

    private int getCode() {
        return code;
    }
}
```